modules/DMZ/models/model_properties.py

```diff
@@ -0,0 +1,10 @@
+# This file will be part of forcing models to have specific properties that govern
+# their usage.
+
+
+class ModelProperties(object):
+
+    def __init__(self, regression=False, online=False):
+        self._classification = not regression  # I don't know if I should get fancy...
```

Comment:
No not an XML file, although that too should be flushed out. I mean inside the class, how we are going to organize things? and find same way to prevent them from being editable?